backend/core/models.py

```
@@ -678,33 +643,171 @@ def update_library(self) -> Union[str, None]:
+        new_dependencies_urn = dependencies_urn - old_dependencies_urn
+
+        if not dependencies_urn.issuperset(old_dependencies_urn):
+            return "invalidLibraryUpdate"
+
+        new_dependencies = []
+        for new_dependency_urn in new_dependencies_urn:
+            try:
+                new_dependency = LoadedLibrary.objects.filter(
+                    urn=new_dependency_urn
+                ).first()  # The locale is not handled by this code
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)